### ISSUE TITLE
Doesn't work with custom ts paths

### ISSUE THREAD

**Issue by kirankunigiri** [sapphi-red/typescript-plugin-toml#2]
When using custom paths with the tsconfig and using them to import the toml file, the typescript server crashes.
I have created a minimal project to replicate the issue here: https://github.com/kirankunigiri/toml-plugin-issue.

Example tsconfig.json paths
```
"baseUrl": ".",
"paths": {
    "@src/*": [
        "src/*"
    ]
},
```

This will work:
```ts
import test from "./test.toml";
```

This will crash the TS server:
```ts
import test from "@src/test.toml";
```

**Comment by sapphi-red** [comment]
I added a fix in 0.4.0. It should work now.

**Comment by kirankunigiri** [comment]
Thanks! It's working now in my project.
